georiviere/flatpages/static/js/tinymce.js

```diff
@@ -1,8 +1,67 @@
-tinyMCE.init({
+tinymce.init({
```

Comment:
It seems curious that this config works with the difference of case

Reply:
```suggestion
tinyMCE.init({
```